crates/proto/src/rr/rdata/svcb.rs

```diff
@@ -270,7 +270,7 @@ impl BinEncodable for SvcParamKey {
 
 impl fmt::Display for SvcParamKey {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> Result<(), fmt::Error> {
-        let mut write_key = |name| write!(f, "{}", name);
+        let mut write_key = |name| f.write_str(name);
```

Comment:
IMO this named closure is adding indirection at ~no benefit, since `write_key` is only two characters more than `f.write_str`.

You could also just keep the `match` statement as the final expression of this block and remove all the try operators from it, since the return value is the same either way.

Reply:
good call, I didn't even notice that.